pkg/apis/autoscaling/v1/conversion.go

```diff
@@ -171,7 +171,8 @@ func Convert_autoscaling_ExternalMetricStatus_To_v1_ExternalMetricStatus(in *aut
 		out.CurrentValue = *in.Current.Value
 	}
 	if in.Current.AverageValue != nil {
-		out.CurrentAverageValue = in.Current.AverageValue
```

Comment:
this is a surprise when i walk thru all the autoscaling conversion. should do a deep copy here.

Reply:
conversion is not required to make copies of the input object values

Reply:
edit: mmm.. quantity might be special.. we don't actually modify that in-place

Reply:
> if we call this conversion directly (not recommended tho), modifying the input obj will propagates to the output. is that okay?

Yes. Generated conversions share memory between the input and output object where possible:

https://github.com/kubernetes/kubernetes/blob/a0c2788249ae1582d10089e7a34bb54fc6b3879d/pkg/apis/core/v1/zz_generated.conversion.go#L7457-L7494